src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1104,7 +1104,7 @@ internal enum ErrorCode
         ERR_VarargsAsync = 4006,
         ERR_ByRefTypeAndAwait = 4007,
         ERR_BadAwaitArgVoidCall = 4008,
-        // ERR_MainCantBeAsync = 4009,
```

Comment:
The convention, at least as well as I can surmise, is to comment out when there is a gap in the ordering.  For instance ERR_BadAwaitWithoutAsyncAnonMeth (20-ish lines down).  It was removed but the error slot 4035 still exists.  In this case we're replacing an error hence 4009 is not a gap anymore.  

This convention makes it easy to find places to put new errors.  Vs having to scan through te file looking for subtle changes in the number. 

---
In reply to: [114816031](https://github.com/dotnet/roslyn/pull/19093#discussion_r114816031) [](ancestors = 114816031)